src/build.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const fs = require('fs');
+const template = require('./template');
+
+function deleteKeys(obj, unwantedKeyArray) {
```

Comment:
should this `deleteKeys` function be in its own file (with exports) that could be utilised in this build.js? Thinking about extensibility and maintainability/testing.

maybe have a `lib` directory within the `src` directory. `lib` directory being for the file that houses the `deleteKeys` function.

Plus this would avoid having to have that package that creates those "magic" getters and setters.